pkgs/top-level/python-packages.nix

```diff
@@ -31346,6 +31346,17 @@ EOF
 
   distro = callPackage ../development/python-modules/distro { };
 
+  smart_open = callPackage ../development/python-modules/smart_open.nix { };
+  
+  serpent = callPackage ../development/python-modules/serpent.nix { };
+
+  pyro4 = callPackage ../development/python-modules/pyro4.nix { };
+
+  pyemd = callPackage ../development/python-modules/pyemd.nix { };
+
+  morfessor = callPackage ../development/python-modules/morfessor.nix { };
+
+  gensim = callPackage ../development/python-modules/gensim.nix { };  
```

Comment:
even though the order of the attributes in this file is a mess, do try to stick to it.